jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1806,7 +1806,7 @@ code_blockt java_bytecode_convert_methodt::convert_instructions(
     // (e.g. due to exceptional control flow)
     if(!start_new_block)
       start_new_block=address_pair.second.predecessors.size()>1;
-    // Start a new lexical block if we've just entered a try block
+    // Start a new lexical block if we've just entered a handler block
```

Comment:
`try` was accurate, I'd interpret `handler` to mean a catch block, and the exception table's `start_pc` entry refers to the `try`

Reply:
I thought using the word "try" would imply the whole of a Java `try` block to someone who didn't know how exception tables worked.

Reply:
Ah, true enough -- in that case I think you should say it in full, "if we've just entered a block in which exceptions are handled. This is usually the start of a `try` block, but a single `try` can be represented as multiple non-contiguous blocks in the exception table."